src/CleanRegex/Internal/Match/Details/MatchDetail.php

```diff
@@ -124,7 +124,7 @@ public function matched($nameOrIndex): bool
     }
 
     /**
-     * @return string[]
+     * @inheritDoc TODO: Changed; confirm
```

Comment:
Kind request to check all `TODO: Changed; confirm` comments and approve the change. I don't intend to introduce a single TODO/FIXME which would not be cross-referenced in a GitHub issue.